### DISCUSSION TITLE
Spor salonu sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#273] [Zeka Sorusu]
Bir spor salonunda üç farklı spor yapılabilmektedir. Bunlar, basketbol, voleybol ve yüzmedir. Spor salonundaki toplam 200 kişinin her biri en az bir sporu düzenli yapmaktadır. Yüzen kişi sayısı, voleybol oynayan kişi sayısı ve basketbol oynayan kişi sayısı aynıdır ve 80’e eşittir. Yüzen ve basketbol oynayan kişi sayısı üç sporu da yapan kişi sayısının iki katıdır. İki farklı sporla ilgilenen kişi sayısı herhangi iki spor için aynıdır.

Soru 1 : Her üç sporu da yapan kaç kişi vardır?

Soru 2 : Voleybol oynayan fakat basketbol oynamayan kaç kişi vardır?

**Comment by akgembi12** [comment]
3 sporu yapan 5 kişi
Voleybol oynayıp basketbol oynamayan 65 Kişi
![16948094596157250669647182651695](https://github.com/sonsuzus/sonsuzus/assets/86284210/50c10d3b-08c4-40a6-9c59-53e690d5c69f)
